.gitignore

```diff
@@ -13,6 +13,7 @@ develop-eggs
 .installed.cfg
 lib
 lib64
+.idea*
```

Comment:
As @mboersma has noted on other PRs, I would suggest this be a separate PR, just to keep is tractable.